app/src/pages/playground/PlaygroundOutput.tsx

```diff
@@ -125,6 +140,8 @@ export function PlaygroundOutput(props: PlaygroundOutputProps) {
       );
     }
     return "click run to see output";
+    // eslint-disable-next-line react-compiler/react-compiler
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [hasRunId, instance.output, instanceId, runId]);
```

Comment:
this is hacky, but does work we have to remove streaming from deps array otherwise when you toggle it it will switch and rerun the mutation or stream (whichever you're switching to)